CONTRIBUTING.rst

```diff
@@ -39,13 +42,56 @@ notes.
 If your change addresses an Issue
 ---------------------------------
 
-Bug fixes are always welcome, especially if they are against known 
+Bug fixes are always welcome, especially if they are against known
 issues!
 
-When you send a pull request that addresses an issue, please add a 
+When you send a pull request that addresses an issue, please add a
 note of the format ``Fixes #24`` in the PR so that the PR links back
 to its relevant issue and will automatically close the issue when the
 PR is merged.
 
 .. _ev3dev: http://ev3dev.org
 
+Building and testing changes on the EV3
+---------------------------------------
+
+In an SSH Terminal window with an EV3 with Internet access,
```

Comment:
super minor...would lowercase the T in terminal

Reply:
Done (or is it done :-)